website/content/docs/commands/web.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+page_title: Web REPL - Command
+description: |-
+  The Vault web user interface (UI) includes an advanced mode that mimics some basic commands from the Vault CLI. It can be useful a user is more familiar with the CLI or API as it lets a user directly input the paths they wish to manipulate.
+
+---
+
+
+# Web REPL
+
+The Vault web user interface (UI) includes an advanced mode that mimics some basic commands from the Vault CLI. It can be useful a user is more familiar with the CLI or API as it lets a user directly input the paths they wish to manipulate.
```

Comment:
```suggestion
The Vault web user interface (UI) includes an advanced mode that mimics some basic commands from the Vault CLI. It can be useful for users more familiar with the CLI or API as it lets a user directly input the paths they wish to manipulate.
```